### ISSUE TITLE
[UI] Maximize panel doesn't work

### ISSUE THREAD

**Issue by pgilfernandez** [olive-editor/olive#1403]
**Commit Hash** 
2c4f781e

**Platform** 
macOS 10.14.6

**Summary**
This feature seems not to work at all under macOS. I have tried via menu Window/Maximize panel and via shortcut and it does nothing. Launching Olive with Terminal doesn't change anything and there is no warning or error logged.

**Additional Information / Output**

**Comment by itsmattkc** [comment]
Works for me on macOS. How are you using it? It maximizes the panel that your cursor is currently hovered over.

**Comment by pgilfernandez** [comment]
OK, I got it to work, thanks.

The problem was due to:

- I thought it maximized the active panel (the one that has a blue border)
- the default keyboard shortcut, "`", is not present in Spanish ISO keyboard (at least in my macbook Pro) so I had to manually set a key that wasn't used by another command.
- **most important**: activating it from menu has no effect (or does not work) because the mouse pointer is not at any panel but in the menu itself. I guess this may be improved: something like, when activating it from menu it will maximize the active panel...
